xarchain/assetlist.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "xarchain",
+  "assets": [
+    {
+      "description": "stXAR is the native cryptocurrency of the Arcana app chain, designed to orchestrate bridge-less interoperability between multiple blockchains",
+      "extended_description": "stXAR powers the Arcana app chain by supporting network consensus through staking, ensuring security and trustless coordination across chains.",
+      "denom_units": [
```

Comment:
it is required to have a denom_unit with exponent 0 (that unit must also be used for the `base` property.